gsoft-inc/wl-orbiter
# [DS-416] Migrate to pnpm

Issue: https://workleap.atlassian.net/browse/DS-416

## Summary

To follow the stack of the Workleap organization. We are migrating from yarn to pnpm as a package manager to improve performance as a whole. Also migrating from lerna and npm-run-all to pnpm only.

## How to test

See if it still deploys on Netlify, Chromatic and passes the CI

- Is this testable with Jest or Chromatic screenshots? no

- Does this need an update to the documentation? yes
